code/src/java/pcgen/io/IIFNode.java

```diff
@@ -31,15 +32,15 @@
  */
 class IIFNode
 {
-	private List<Object> falseChildren;
-	private List<Object> trueChildren;
-	private String expr;
+	private final List<Object> falseChildren;
+	private final List<Object> trueChildren;
+	private final String expr;
 
-	IIFNode(String _expr)
+	IIFNode(final String expr)
```

Comment:
understood.  I don't know the pcgen style so I'm proposing changes and asking for feedback :-)